Updated mmctl export create command flag (#25474)
mattermost/mattermost: server/cmd/mmctl/docs/mmctl_export_create.rst

::

  -h, --help                        help for create
      --include-archived-channels   Include archived channels in the export file.

>       --no-attachments              Omit file attachments in the export file.

Author: ```suggestion
      --no-attachments              Exclude file attachments from the export file.
```
